Bugzilla/Bug.pm

```diff
@@ -4180,7 +4180,7 @@ sub groups {
       . " THEN 1 ELSE 0 END,"
       . " CASE WHEN groups.id IN($grouplist) THEN 1 ELSE 0 END,"
       . " isactive, membercontrol, othercontrol"
-      . " FROM groups"
+      . " FROM " . $dbh->quote_identifier('groups')
```

Comment:
IMO, if we're already pretty MYSQL-specific, it might be simpler and easier to read to manually quote this eh?
```
...
. ' FROM "groups"'
...
```

I'm guessing that `dbh->quote_identifier()` is still a good idea if the table name comes from a variable, though.

What do you think?

Reply:
PostgreSQL uses double quotes instead and I do not want to knowingly break Pg. Using ->quote_identifier allows for any other changes needed in the future.